src/server/lib/logger.ts

```diff
@@ -1,34 +1,45 @@
 import { Logger, LogLevel } from '@/server/models/Logger';
 import { createLogger, transports } from 'winston';
+import * as util from 'util';
```

Comment:
```suggestion
import { format } from 'util';
```

Just for a mini win on performance?